scripts/identities.txt

```diff
@@ -0,0 +1,10 @@
+tz1L7zaWD1aRYBTQvSdxEdc9KDzfwG4DydDu edsk4WVyqJ112qJREwC5TERn2AmmmNMyVhwWyaLgHosC48UFV6EnMG
+tz1LKyW7AzoXYiPxxS6uzKwcUtpincBZduxf edsk4JaFU4PZ7vpp1kDyeE578zwJv2XpBP7QqhSQEw3mFS3rmBP5nJ
+tz1RPNjHPWuM8ryS5LDttkHdM321t85dSqaf edsk36FhrZwFVKpkdmouNmcwkAJ9XgSnE5TFHA7MqnmZ93iczDhQLK
+tz1SZtpiT7KAdwZ23PDWDwdrkbe1Ci6Zzb48 edsk3Vjga1rxDHC9Bns7EN4CWkpaYoHwnHBPJMdERPtytxcx7A78PG
+tz1UgD3EScpgxVNVFQ3BKWspGMBzjCAUBK8z edsk3tP2tJieBNsRTqaq8edcfekPxALp3W5F2Ef4tLecqf6XG66m5F
+tz1VA8z9thUhqJDq6hFnD1V5FNfNPzeuQd61 edsk4Jkbb34LLESFeXdeNdwKhuaVarxT8QwRzerf5V9oqbjrjGAzNx
+tz1X7hncSDZkz5Xj2Yb4a4rFwmdh2vnf1Xcn edsk37WmmUhP2g5AN9Jz5FWgcspMyYbsqzrGW7bgUXMRrEd4M9aTaG
+tz1cQtKosPZ7rZNTpinqRXGxjC9HKUJzYiDC edsk44pF6zGm8qV4YShrABDZDq5jrGydwrWmMiEwJWsJEarMSwCmB4
+tz1cXjXadqbG4Pp3DFZJETi3vCPwBM77LBpi edsk3MG557SuyCMyyjBa6n9MeZsLGRkHnXPpJedu8UrbUoEFh5q64K
+tz1iPt3doWgCDrPDw49hRD47zvuaE1VKStFU edsk33pagYM8CfcD2gokyE7RWD5bjcRAqdoYjLDuMohg5yT9r6UfYY
```

Comment:
We tend to use key hashes and secrets which are not explicitly associated one to another, which, in turn, makes refactoring harder.